lib/events/event_handler.ex

```diff
@@ -78,7 +74,17 @@ defmodule ElixirTools.Events.EventHandler do
   def publish_event_call(event, opts) do
```

Comment:
```suggestion
  defp publish_event_call(event, opts) do
```

Reply:
I'll change, butt that is out of the scope of this PR and in any case is code to be removed when all the events are migrated to the new flow.